libs/shared/components/src/auth/UserMenu/UserDelegations.tsx

```diff
@@ -72,48 +62,49 @@ export const UserDelegations = ({
     )
   }
 
-  // No data.
-  if (delegations.length === 0 && !loading && !error) {
-    return null
-  }
-
   const onClickDelegation = (delegation: Delegation) => {
     onSwitchUser(delegation.nationalId)
   }
-
   return (
-    <>
-      <hr className={styles.hr} />
-      <Text variant="h5" as="h5" marginBottom={2}>
+    <Box>
+      <Text variant="small" marginBottom={1}>
         {formatMessage(userMessages.delegationList)}
       </Text>
-      <List>
-        {delegations.map((delegation) => (
-          <TopicCard
-            key={delegation.nationalId}
-            size="small"
-            tag={
-              delegation.isCurrent
-                ? formatMessage(userMessages.selectedDelegation)
-                : undefined
-            }
-            onClick={
-              delegation.isCurrent
-                ? undefined
-                : () => onClickDelegation(delegation)
-            }
-          >
-            {delegation.name || delegation.nationalId}
-          </TopicCard>
-        ))}
-        {loading ? (
-          <SkeletonLoader display="block" height={59} borderRadius="large" />
-        ) : error ? (
-          <Text color="red400">
-            {formatMessage(userMessages.delegationError)}
-          </Text>
-        ) : null}
-      </List>
-    </>
+      <Box className={styles.userDelegationWrapper}>
+        <Stack space={1}>
+          {!!actor && (
+            <UserTopicCard
+              colorScheme={'purple'}
+              onClick={() => onSwitchUser(actor?.nationalId)}
+            >
+              {actor?.name}
+            </UserTopicCard>
+          )}
```

Comment:
Design comment:

Not loving the design of the delegation list during an active delegation.

* The purple card background is so light I can barely see it, and I think it's weird that the Avatar circle disappears into it.
* The active delegation is still in the list, but shows no special state indicating that it is active and can't be clicked.

![image](https://user-images.githubusercontent.com/115094/146916930-d69b994a-925a-459c-87fd-a2901401554a.png)

Reply:
Whops - The current user should not be in the list anymore and the avatar should be different color - thanks I'll fix it!
But the purple color is to show you your main user. I misunderstood it when we spoke the other day. The heading displaying the current user and main user below is suppose to be enough to show active delegation but i'll take this one to the designer as well.

Reply:
Ok sure. Maybe the fixes you mention will be enough for it to make more sense to me.

Reply:
Done ✅ Fixed avatar background color and removed current actor from list.